h/templates/accounts/signup-successful.html.jinja2

```diff
@@ -0,0 +1,15 @@
+{% extends "h:templates/accounts/base.html.jinja2" %}
+
+{% block page_title %}{{ heading|default("Sign up for Hypothesis") }}{% endblock %}
+
+{% block form_message %}
+  {% if not form %}
+    {% if message %}<p>{{ message }}</p>{% endif %}
+
+    <p>Please check your email and open the link to activate your account. If you
+    don't receive your activation link within a few minutes, send an email to
+    <a href="mailto:support@hypothes.is">support@hypothes.is</a></p>
```

Comment:
```suggestion
    <p>
        Please check your email and open the link to activate your account.
    </p>
    <p>
        If you don't receive your activation link within a few minutes, send an email to
        <a href="mailto:support@hypothes.is">support@hypothes.is</a>
    </p>
```

We should separate the action as a single line.

Reply:
Done: https://github.com/hypothesis/h/pull/5964/commits/678c229695fcb0d1046a636043dfb73197cfa6a2